external/storm-kafka/src/jvm/org/apache/storm/kafka/bolt/KafkaBolt.java

```diff
@@ -51,7 +51,9 @@
  * This bolt uses 0.8.2 Kafka Producer API.
  * <p/>
  * It works for sending tuples to older Kafka version (0.8.1).
+ * @deprecated Please use the KafkaBolt in storm-kafka-client
  */
+@Deprecated
```

Comment:
Maybe it would be fine to just remove it here immediately and deprecate it in 1.1.0?